bin/dev

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+npm start
```

Comment:
@smaboshe I'm not sure I understand the purpose of this file? How is this simpler than running `nom start`?

If there's something I am missing and this has some advantages it should be documented in the README.

Reply:
Thanks, @LkeMitchll. It's more of a convention that we use on Rails projects. I added this to `design-dot` and since we will be merging the two thought this would be good to add here as well. `npm start` will still work. I also don't mind rolling these changes back if keeping `npm start` is more familiar 🙂.

I'll update the README if you don't mind keeping it and will the change back if that works better.